internal/lexer/items/item.go

```diff
@@ -34,15 +34,9 @@ type Item struct {
 	*symbols.Symbols
 }
 
-/*
-For a lex production,
-  T : s
-func NewItem returns
-  T : •s
-without executing the ℇ-moves for s.
-
-Func *Item.Emoves must be called to return the set of basic items for T : •s.
-*/
+// NewItem will return `T : •s` for a lex production `T : s` without executing
```

Comment:
It is pretty nice to start with `NewItem`, but otherwise it being broken up over more lines and inside `/*` comment was slightly more readable.